Makefile.am

```diff
@@ -58,6 +58,7 @@ install-data-hook:
 hanoarchdir = $(datadir)/@PACKAGE@/hb_report
 hanoarch_DATA = hb_report/constants.py hb_report/utillib.py
 hanoarch_SCRIPTS = hb_report/hb_report
+var_log_crmsh = $(DESTDIR)/${localstatedir}/log/crmsh
```

Comment:
We don't need to define it here. Be aware the prefixes and suffixes of these variables are meaningful and imply certain processing done by automake/make.